srcpkgs/pipewire/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'pipewire'
 pkgname=pipewire
 version=1.0.3
-revision=1
+revision=2
 build_style=meson
 configure_args="
  --auto-features=enabled
  -Db_ndebug=false
  -Ddocs=enabled
  -Dman=enabled
- -Dlibcamera=disabled
+ -Dlibcamera=enabled
```

Comment:
Just delete the line altogether, the option is auto and will pick up the dependency when present.

Reply:
Explicit instead of auto has the advantage that the build will fail if the dep can't be found for any reason.

Reply:
Fair enough, for something like this it should probably stay.